ui/raidboss/data/03-hw/raid/a2s.js

```diff
@@ -154,18 +154,18 @@
     },
     {
       'locale': 'fr',
-      'missingTranslations': true,
       'replaceSync': {
+        'Bomb': 'Bombe',
         'Boomtype Magitek Gobwalker G-VII': 'gobblindé magitek G-VII Boumbardier',
-        'Giant Bomb': 'Atout',
-        'Gordian Hardhelm': 'casque-dur gordien',
-        'Gordian Hardmind': 'cerveau-dur gordien',
-        'Gordian Sniper': 'sniper gordien',
-        'Gordian Soldier': 'soldat gordien',
-        'Hangar 12': 'Grand hangar GH-12',
-        'Jagd Doll': 'poupée jagd',
-        'King Gobtank G-IV': 'gobtank G-IV roi',
-        'Magitek Gobwidow G-IX': 'gobmygale magitek G-IX',
+        'Giant(?! Bomb)': 'Bombe géante',
```

Comment:
I think this should be `'Giant Bomb'` and `'Bomb'` should be `'(?!Giant )Bomb'`

Reply:
I trust you.

Reply:
What I did worked after testing games. I tried what you told me, but it gives an error. so I reinstated.
![a2s](https://user-images.githubusercontent.com/61565834/82534873-3b1d1880-9b46-11ea-940a-a1ff39ed4e7c.PNG)

Reply:
`'Giant(?! Bomb)': 'Bombe géante'` will translate "Giant Bomb" into "Giant Bombe géante", which seems like not what you want.

In that case, `'Giant Bomb'` should just be `'Giant Bomb'` and `'Bomb'` should be `'(?<!Giant )Bomb(?!e)'`

Reply:
Oh ok. Thank you.

Reply:
`(?<! )Bomb(?!e)': 'Bombe',`
Wouldn't it be better?

Reply:
Sure, I think that's equivalent.